src/App.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { useEffect, useState } from "react";
+import { getProducts } from "./api/api";
+import Header from "./component/Header";
+import AllProduct from "./component/AllProduct";
+import BestProduct from "./component/BestProduct";
+// import Market from "./pages/Market";
+// import Dropdown from "./component/Dropdown";
+// import SortIcon from "./images/ic_sort.svg";
+// import SearchIcon from "./images/ic_search.svg";
+
+const LIMIT = 6;
+
+function App() {
+  const [order, setOrder] = useState('createAt');
+  const [items, setItems] = useState([]);
+  const [bestItem, setBestItem] = useState([]);
+  const [offset, setOffset] = useState(0);
+  const [dropdownOpen, setDropdownOpen] = useState(false);
+
+  const sortedItems = [...items].sort((a, b) => b[order] - a[order]);
+  const BestProductItem = bestItem.slice(0, 4);
+
+  const toggleDropdown = () => setDropdownOpen(!dropdownOpen);
+
+  const handleLoad = async (options) => {
+    const { list } = await getProducts(options);
+    if (options.offset === 0) {
+      setItems(list);
+    } else {
+      setItems([...items, ...list]);
+    }
+    setOffset(options.offset + list.length);
+    setBestItem(list);
```

Comment:
- if/else문 대신 삼항연산자를 활용하면 더 간결하게 표현해볼 수 있을 것 같습니다.
- 베스트 상품이 단순히 list 데이터의 앞부분만을 자른 것이라면, 굳이 bestItem state를 따로 만들 필요는 없어보입니다.
```suggestion
    setItems(options.offset === 0 ? list : [...items, ...list]);
    setOffset(options.offset + list.length);
```